gensim/test/test_ldamodel.py

```diff
@@ -251,20 +251,31 @@ def testShowTopics(self):
 
 
     def testGetDocumentTopics(self):
-        doc_topics = self.model.get_document_topics(self.corpus)
+
+        numpy.random.seed(0)
+        model = self.class_(self.corpus, id2word=dictionary, num_topics=2, passes= 100)
+
+        doc_topics = model.get_document_topics(self.corpus)
 
         for topic in doc_topics:
             self.assertTrue(isinstance(topic, list))
             for k, v in topic:
                 self.assertTrue(isinstance(k, int))
                 self.assertTrue(isinstance(v, float))
 
-        doc_topics = self.model.get_document_topics(self.corpus[0])
+        doc_topics, word_topics = model.get_document_topics(self.corpus[0], per_word_topics=True)
 
         for k, v in doc_topics:
             self.assertTrue(isinstance(k, int))
             self.assertTrue(isinstance(v, float))
 
+        for w, t in word_topics:
+            self.assertTrue(isinstance(w, int))
+            self.assertTrue(isinstance(t, int))
+
+        # first word in the doc belongs to topic 2
+        self.assertEqual(word_topics[0], (0, 1))
```

Comment:
Tuple `(0,1)` basically means 0th word corresponds to 1st topic. 
Should I change it to `word_topics[0][1]` equals to 1?